gateway/ln-gateway/src/lib.rs

```diff
@@ -372,6 +365,7 @@ impl Gateway {
                 move |handle| async move {
                     loop {
                         if handle.is_shutting_down() {
+                            info!("We are shutting down, exiting from loop...");
```

Comment:
Nit: Maybe a more description message like "Gateway HTLC handler loop is shutting down"